test/acceptance/cli-test.js

```diff
@@ -319,40 +382,64 @@ describe('ember-template-lint executable', function() {
       });
 
       describe('given a directory with errors and a lintrc with rules', function() {
-        it('should print properly formatted error messages', function() {
-          let result = run(['.', '--config-path', '../with-errors/.template-lintrc'], {
-            cwd: './test/fixtures/without-errors',
+        it('should print properly formatted error messages', async function() {
+          setProjectConfigWithoutErrors();
+
+          let projectWithErrors = new Project();
+          projectWithErrors.chdir();
+          projectWithErrors.setConfig({
+            rules: {
+              'no-bare-strings': true,
+            },
+          });
+
+          let result = run(['.', '--config-path', projectWithErrors.path('.template-lintrc.js')], {
```

Comment:
Since you are project.chdir'ing you don't need to pass `--config-path` (and the test isn't _specifically_ about passing a custom config path)

Reply:
my bad, i shouldn't be changing the project dir. `chdir` shouldn't be here at all since its only used to reference the `.template-lintrc.js` in `--config-path`